p/themes/base-theme/template.css

```diff
@@ -1187,7 +1187,7 @@ br {
 .nav label {
 	padding: 0;
 	display: none;
-	width: 10%;
+	width: 65px;
```

Comment:
65 on purpose instead of 64?

Reply:
yes. I like numbers that ends on 0 or 5, if there is not a very strong reason to use another one.

Reply:
The reason is that 64 = 4 * 16. I don't have some weird attachment to that but it is the logical value. ;-)